lib/entity/fields/entity-binary-field.ts

```diff
@@ -0,0 +1,38 @@
+import { EntityField } from "./entity-field";
+import { RedisHashData, RedisJsonData } from "../../client";
+import { EntityValue } from "../entity-value";
+
+export class EntityBinaryField extends EntityField {
+  toRedisJson(): RedisJsonData {
+    const data: RedisJsonData = {};
+    if (this.value !== null) data[this.name] = this.valueAsBuffer.toString('base64')
```

Comment:
should be `data[this.name] = [...this.valueAsBuffer]` instead (store it as array of numbers)